pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -2556,3 +2556,8 @@ SELECT timezone('1970-01-01 01:00'::timestamptz, 'UTC+6')
 
 statement ok
 SET TIME ZONE +0
+
+query T
+SELECT getdatabaseencoding()
```

Comment:
can you give this an example of moving to a different encoding and checking it works as expected?

Reply:
I have tried to create, test case for different encoding but not sure how I could create database with different encoding.

Reply:
according to the docs, you can use `CREATE DATABASE ... ENCODING = x` (see https://www.cockroachlabs.com/docs/stable/create-database.html)

so create that database, use that database, check encoding is same, and ideally reset state back to the original database used (i think it's `test`).

can you also add `subtest getdatabaseencoding` above `query T` to add some sort of separation? thanks!

Reply:
sure, and thanks

Reply:
looking at the docs again, we only support utf-8 encoding anyway.

Reply:
yes, I am unable to create databases having encoding other than 'UTF-8'